addons/website/static/src/js/content/website_root.js

```diff
@@ -187,13 +188,8 @@ var WebsiteRoot = publicRootData.PublicRoot.extend(KeyboardNavigationMixin, {
         document.body.classList.add('o_fullscreen_transition');
         document.body.classList.toggle('o_fullscreen', this.isFullscreen);
         document.body.style.overflowX = 'hidden';
+        this.trigger_up('adjust_iframe');
```

Comment:
Don't trigger_up an event that is meant to be caught by the same object only. Instead create a private method and call it directly (here and in your event handler)

Reply:
yes, we can directly call `_onAdjustIFrame` here, but I don't understand for event handler ?